website/content/api-docs/secret/identity/mfa/duo.mdx

```diff
@@ -0,0 +1,159 @@
+---
+layout: api
+page_title: /identity/mfa/method/duo - HTTP API
+description: >-
+  The '/identity/mfa/method/duo' endpoint focuses on managing Duo MFA behaviors in Vault.
+---
+
+## Configure Duo MFA Method
+
+This endpoint defines an MFA method of type Duo.
+
+| Method | Path                           |
+| :----- |:-------------------------------|
+| `POST` | `/identity/mfa/method/duo/:id` |
+
+### Parameters
+
+- `id` `(string: "")` - Optional UUID to specify if updating an existing method.
+
+- `mount_accessor` `(string: <required>)` - The mount to tie this method to for use in automatic mappings. The mapping will use the Name field of Aliases associated with this mount as the username in the mapping.
+
+- `username_format` `(string)` - A format string for mapping Identity names to MFA method names. Values to substitute should be placed in `{{}}`. For example, `"{{alias.name}}@example.com"`. If blank, the Alias's Name field will be used as-is. Currently-supported mappings:
```

Comment:
I'm not really sure what `MFA method names` means in this context, I know we're using ids not names on the `identity/mfa/method..` endpoints. My understanding is that it maps to the username of the MFA user on the actual MFA setup they're using, so like okta email, duo name etc. Would maybe a couple words around what `mfa method names` means be helpful to disambiguate that ?
( also should the thing that `username_format` maps to, be different for the different auth types ? like for okta its an email, for duo i assume something else etc. )

Reply:
As for the second question, yes. As far as I know, okta works primarily with emails, but duo accepts common usernames (like the ones we configure in userpass mount in vault) as well.

Reply:
gotcha, thnx for the clarification!

Reply:
I'll be honest - I copied most of the verbiage for Duo/PingID/Okta/TOTP from the existing MFA documentation, since the majority of the functionality is identical. I see what you mean though. I'll try to disambiguate this as best I can.

Reply:
that'd be awesome! yeap i saw this was also the language in the existing sys/mfa docs, i had the same confusions on those too tbh so figured i'd ask the q here.